subcommands/waves/cmd.go

```diff
@@ -12,7 +12,7 @@ var api *client.Api
 var cmd = &cobra.Command{
 	Use:     "waves",
 	Aliases: []string{"wave"},
-	Short:   "Manage factory's waves",
+	Short:   "Manage a Factory's Waves",
```

Comment:
It looks like this is the only instance of using `Factory's` in Fioctl.
It makes me regret of my frivolous language, and begging to, maybe, just write `Factory Waves`.